git-find-lfs-extensions.py

```diff
@@ -67,34 +48,34 @@ def add_file(ext, size_mb):
         result[ext]['min'] = size_mb
 
 def print_line(ext, share_large, count_large, count_all, size_all, min, max):
-    print '{}{}{}{}{}{}{}'.format(
+    print ('{}{}{}{}{}{}{}'.format(
         ext.ljust(COLUMN_WIDHT),
         str(share_large).rjust(10),
         str(count_large).rjust(10),
         str(count_all).rjust(10),
         str(size_all).rjust(10),
         str(min).rjust(10),
         str(max).rjust(10)
-    )
+    ))
 
-for root, dirs, files in os.walk(cwd):
+for root, dirs, files in os.walk(os.getcwd()):
```

Comment:
That's a matter of taste. Reviewing my changes I realized https://github.com/larsxschneider/git-repo-analysis/blob/master/git-find-lfs-extensions.py#L85 should have been adapted as well.